cmd/auth-provider-gcp/provider/provider_test.go

```diff
@@ -85,6 +84,7 @@ func TestContainerRegistry(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error while getting response: %s", err.Error())
 	}
+
```

Comment:
check both `hasURL(strings.Split(dummyImage, "/")[0])` and `hasURL("container.cloud.google.com")`